src/main.rs

```diff
@@ -206,11 +246,30 @@ fn entry() -> Result<(), String> {
                 }
             });
 
-            // Print the remaining tags. The `unwrap` is safe assuming no poisoning.
-            for tags in tags_map.lock().unwrap().values() {
-                for tag in tags {
-                    println!("{}", tag);
-                }
+            // Get the remaining tags. The `unwrap` is safe assuming no poisoning.
+            let mut references: Vec<_> = tags_map
+                .lock()
+                .unwrap()
+                .values_mut()
+                .flat_map(|tags| {
+                    tags.iter_mut().map(|tag| {
+                        vec![
+                            std::mem::take(&mut tag.label),
+                            format!("{}:{}", tag.path.display(), tag.line_number),
+                        ]
+                    })
+                })
+                .collect();
+            references.sort();
+
+            // Create a 2-column table of 80 characters length.
+            let mut colonnade = Colonnade::new(2, 80).unwrap();
+
+            colonnade.columns[0].alignment(Alignment::Left);
+            colonnade.columns[1].alignment(Alignment::Left);
+
+            for line in colonnade.tabulate(references).unwrap() {
+                println!("{line}");
             }
```

Comment:
All this colonnade logic seems to be duplicated. Can we pull it out into a reusable function?